### ISSUE TITLE
SVG not loading in final website

### ISSUE THREAD

**Issue by DaBlazePRO** [526avijitgupta/gokarna#88]
Hi I still need a bit of assistance with this, by the way this is my website: https://cloudy-fi.com/

I just need to know why they are being 404'd as mentioned before and what i can do to fix it

**Comment by sebbemunich** [comment]
True. Locally it all works fine. However permissions / owner on the hoster are the same for /images and /icons. Still can´t pinpoint it down.

**Comment by yashmehrotra** [comment]
@sebbemunich You can try GitHub or Netlify hosting

**Comment by jamesericdavidson** [comment]
Bear in mind that hardened Firefox configurations deliberately fail to render SVG files:

`user_pref("svg.disabled", true);`

But I also suspect your problem exists on the host end.

**Comment by CodePurble** [comment]
For me, the SVGs for socials don't render anywhere (local or GitHub Pages). You can find the sources [here](https://github.com/CodePurble/blog). The example site works even if build locally, though. Any help?
